### ISSUE TITLE
[BUG - Release] CMake Error: "Zydis" that is not in any export set.

### ISSUE THREAD

**Issue by abevol** [UE4SS-RE/RE-UE4SS#100]
**Branch or Release**
commit: a19565b

**Describe the bug**
Failed to generate solution with cmake.

**To Reproduce**
1. Clone the repo and restore submodules.
2. CD to VS_Solution, and run 'generate_vs_solution.bat'.
3. See error in console.

**CMake Log**
```
cmake -DCMAKE_BUILD_TYPE=Release -DRC_FORCE_ALL_STATIC_LIBS= -G"Visual Studio 17 2022" ..
-- The CXX compiler identification is MSVC 19.33.31631.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.33.31629/bin/HostX64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
IPO - Supported: YES;
Project: UE4SS
-- The C compiler identification is MSVC 19.33.31631.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.33.31629/bin/HostX64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
Project: File (STATIC)
Project: DynamicOutput (STATIC)
Project: Function (HEADER-ONLY)
Project: Constructs (HEADER-ONLY)
Project: Helpers (HEADER-ONLY)
Project: SinglePassSigScanner (STATIC)
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Failed
-- Performing Test COMPILER_HAS_DEPRECATED
-- Performing Test COMPILER_HAS_DEPRECATED - Success
Project: ASMHelper (STATIC)
** AsmJit Summary **
   ASMJIT_DIR=D:/workspace/engines/unreal-engine/RE-UE4SS/RE-UE4SS/Dependencies/PolyHook_2_0/asmjit
   ASMJIT_TEST=FALSE
   ASMJIT_TARGET_TYPE=STATIC
   ASMJIT_DEPS=
   ASMJIT_LIBS=asmjit
   ASMJIT_CFLAGS=-DASMJIT_STATIC
   ASMJIT_PRIVATE_CFLAGS=-MP;-GF;-Zc:inline;-Zc:strictStrings;-Zc:threadSafeInit-;-W4;-DASMJIT_STATIC
   ASMJIT_PRIVATE_CFLAGS_DBG=-GS
   ASMJIT_PRIVATE_CFLAGS_REL=-GS-;-O2;-Oi
Project: Unreal (STATIC)
Project: MProgram (HEADER-ONLY)
Project: Input (STATIC)
Project: LuaRaw (STATIC)
Project: LuaMadeSimple (STATIC)
Project: FunctionTimer (STATIC)
Project: ScopedTimer (HEADER-ONLY)
-- Module support is disabled.
-- Version: 9.1.1
-- Build type: Release
-- CXX_STANDARD: 23
-- Required features: cxx_variadic_templates
Project: ParserBase (STATIC)
Project: IniParser (STATIC)
Project: JSON (STATIC)
Project: ArgsParser (STATIC)
Project: LuaWrapperGenerator (STATIC)
[LuaWrapperGenerator] LLVM_PATH: D:\LLVM\14.0.6\llvm+clang-14.0.6-x86_64-windows-msvc-release-mt
UE4SS
Major: 2, Minor: 5, Hotfix: 1, PreRelease: 0, Beta: 0
-- The ASM_MASM compiler identification is MSVC
-- Found assembler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.33.31629/bin/HostX64/x64/ml64.exe
-- Configuring done (7.4s)
CMake Error: install(EXPORT "PolyHook_2-targets" ...) includes target "PolyHook_2" which requires target "Zydis" that is not in any export set.
CMake Error: install(EXPORT "PolyHook_2-targets" ...) includes target "PolyHook_2" which requires target "Zycore" that is not in any export set.
-- Generating done (0.7s)
CMake Warning:
  Manually-specified variables were not used by the project:

    RC_FORCE_ALL_STATIC_LIBS

CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Desktop (please complete the following information):**
 - OS: Win11

**Comment by narknon** [comment]
This should be benign and shouldn't effect compiling, but let us know if there are build issues.

**Comment by abevol** [comment]
> This should be benign and shouldn't effect compiling, but let us know if there are build issues.

Yes, this does not affect the compilation, thanks. By the way, the 'LuaBindingsGenerator' project is missing the 'clang-c' header files and the library file 'libclang.lib'.

**Comment by narknon** [comment]
Right, just build without that if on release and it's removed on Dev branch.

**Comment by Buckminsterfullerene02** [comment]
This has been fixed in #103
